gatsby-config.js

```diff
@@ -13,41 +13,48 @@ const plugins = [
     options: {
       postCssPlugins: [],
       precision: 5
-    },
+    }
   },
   {
     resolve: `gatsby-plugin-page-creator`,
     options: {
-      path: `${PROJECT_ROOT}/src/patternfly`,
+      path: `${PROJECT_ROOT}/src/patternfly`
     }
```

Comment:
Can we address these linting issues with a better linter so your reviews aren't always fixing things?

Reply:
Yeah.  That might be a good idea.  Are you opposed to using eslint with the same config that patternfly-react uses?